nginx/nginx.conf

```diff
@@ -350,7 +350,6 @@ http {
         rewrite ^/discussion        /ui/discussion.html last;
         rewrite ^/folder            /ui/folder.html last;
         rewrite ^/group             /ui/group.html last;
```

Comment:
@sathomas : They should all point to `ui/index.html`(i.e. the static HTML page that corresponds to the path)